--------------------------------------------------------------------------------
2025-02-13, 15:38:34
--------------------------------------------------------------------------------
<class 'TypeError'>: 
'EigerImage' object does not support indexing
--------------------------------------------------------------------------------
  File "/Users/danshim/ASU Dropbox/Sang-Heon Shim/Python/PeakPo/PeakPo-7.8.0/peakpo/control/cakecontroller.py", line 133, in addremove_cake
    update = self._addremove_cake()
  File "/Users/danshim/ASU Dropbox/Sang-Heon Shim/Python/PeakPo/PeakPo-7.8.0/peakpo/control/cakecontroller.py", line 188, in _addremove_cake
    self.process_temp_cake()
  File "/Users/danshim/ASU Dropbox/Sang-Heon Shim/Python/PeakPo/PeakPo-7.8.0/peakpo/control/cakecontroller.py", line 226, in process_temp_cake
    self._load_new_image()
  File "/Users/danshim/ASU Dropbox/Sang-Heon Shim/Python/PeakPo/PeakPo-7.8.0/peakpo/control/cakecontroller.py", line 197, in _load_new_image
    self.model.load_associated_img()
  File "/Users/danshim/ASU Dropbox/Sang-Heon Shim/Python/PeakPo/PeakPo-7.8.0/peakpo/model/model.py", line 385, in load_associated_img
    self.diff_img.load(filen_toload)
  File "/Users/danshim/ASU Dropbox/Sang-Heon Shim/Python/PeakPo/PeakPo-7.8.0/peakpo/ds_cake/DiffractionImage.py", line 33, in load
    data = images[0]
